gapic-generator/test/gapic/presenters/gem/garbage_test.rb

```diff
@@ -40,7 +40,7 @@ def test_garbage
     assert_equal ["endless.trash.forever"], presenter.packages.map(&:name)
     presenter.packages.each { |pp| assert_kind_of Gapic::Presenters::PackagePresenter, pp }
 
-    assert_equal ["GarbageService", "ReallyRenamedService", "ResourceNames", "IAMPolicy"], presenter.services.map(&:name)
+    assert_equal ["GarbageService", "ReallyRenamedService", 'DeprecatedService', "ResourceNames", "IAMPolicy"], presenter.services.map(&:name)
```

Comment:
Google's Ruby style is to use double quotes for strings, even those that don't have any interpolation.

Reply:
Done.